src/errors.rs

```diff
@@ -54,7 +54,7 @@ impl From<anyhow::Error> for CliError {
 
 impl From<clap::Error> for CliError {
     fn from(err: clap::Error) -> CliError {
-        let code = if err.use_stderr() { 1 } else { 0 };
+        let code = i32::from(err.use_stderr());
```

Comment:
I actually dislike that clippy lint and would prefer for it to be marked `allowed`

Reply:
Wait, something is up.  My local copy of this code has an `allow` already in it.  This dates back about a month: 6e5f6b74899bde7c50ecbd35f0294bbf1dd4d20c